cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -82,36 +87,30 @@ describe.skip('FluxQueryBuilder', () => {
       cy.getByTestID('field-selector')
         .should('be.visible')
         .should('not.contain', 'Loading')
-      cy.getByTestID('tag-selector--key')
+      cy.getByTestID('tag-selector-value--header-wrapper')
         .should('be.visible')
         .should('not.contain', 'Loading')
 
-      // search a feild, should contain only the feild, no tag keys
+      // search a field, should contain only the field, no tag keys
       cy.getByTestID('field-tag-key-search-bar')
         .should('be.visible')
         .type(searchField)
       cy.getByTestID('field-selector')
         .should('be.visible')
         .should('not.contain', 'Loading')
         .should('not.contain', 'No Fields Found')
-      cy.getByTestID('tag-selector--key')
+      cy.getByTestID('tag-selector')
```

Comment:
i was having issues here with it not being able to find the Loading text because the tags that show up had a Loading div somewhere in their markup that was "hidden" but cypress was finding them as visible.